src/main/java/RestApi/movies/service/MovieService.java

```diff
@@ -1,7 +1,59 @@
 package RestApi.movies.service;
 
+import RestApi.movies.model.Movies;
+import RestApi.movies.model.Movie;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Component;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 @Component
 public class MovieService {
+
+    ObjectMapper objectMapper = new ObjectMapper();
+
+    public Movies findAll() throws IOException {
+
+        byte[] jsonData = Files.readAllBytes(Paths.get("movies.json"));
+
+            Movies movies = objectMapper.readValue(jsonData, Movies.class);
+            return movies;
+
+    }
+
+    public ResponseEntity<Movie> findMovie(int id)  {
```

Comment:
The service layer is typically reserved for business logic - it shouldn't really have any knowledge of the `ResponseEntity` being returned to the client, the controller should construct that really since its responsibility is to marshal a response.